app/src/main/java/inz/NeuralModel.java

```diff
@@ -188,34 +188,35 @@ public Mat preProcessOneFace(Mat face, Rect detectedFaces)
          */
         public ArrayList<Mat> preProcessAllFaces(Mat frame, MatOfRect detectedFaces)
         {
-            ArrayList<Mat> cuttedFaces = new ArrayList();
+            ArrayList<Mat> cutFaces;
+            cutFaces = new ArrayList();
             for (Rect face : detectedFaces.toArray())
             {
                 Mat faceImg = frame.submat(face);
                 Rect[] eyeArray = findEyesOnImg(faceImg);
                 if(eyeArray.length != 2)
                 {
                     // Sth go wrong, we should stop preprocessing
```

Comment:
no, we don't. Finding eyes on image done by OpenCv works well on very good photo, but one eye closed, glasses or big nose make noise and there is more or less eyes. In such case we should not rotate image, because it is impossible.